components/dashboard/src/workspaces/CreateWorkspacePage.tsx

```diff
@@ -353,7 +353,8 @@ export function CreateWorkspacePage() {
             <div className="flex flex-col max-h-screen max-w-lg mx-auto items-center w-full">
                 <Heading1>New Workspace</Heading1>
```

Comment:
Cross-posting for visibility from Figma some [relevant discussion](https://www.figma.com/file/MLJUI2WQ3ZSZvd1dGhrfbu?node-id=2194:2911#450684931) (internal) about heading spacing here.

> This (**heading padding**) was an after effect from https://github.com/gitpod-io/gitpod/pull/17405/files#diff-36bccdd9dc3633e37674f9a58735d9a42981797a0d200ffc5bcaaf79c3b999d3L22.

> See relevant discussion in https://gitpod.slack.com/archives/C02EN94AEPL/p1683057656072399 (internal).

> The change was driven from the Dedicated admin onboarding but affected all h1 headings across the dashboard.

> Some space between the h1 and the description below would be nice.

> 

> I'm to blame for removing the space between in https://github.com/gitpod-io/gitpod/pull/17553 but having too much space can also make the element feel visually unbalanced.

> See also relevant discussion in https://gitpod.slack.com/archives/C01KPEPQYE6/p1683652843912329 (internal).